binary_transparency/firmware/README.md

```diff
@@ -92,12 +89,21 @@ Planned future enhancements:
 
 Running the Demo
 ----------------
+In running this demo you will take on the role of several different actors within the ecosystem
+to see how making firmware discoverable works in practice. You will then take the role of an
+attacker trying to install malicious code onto a device, and see how the application of
+transparency has made this attack much more expensive.
+
+During the demo flow you will open several different terminals. It's far easier to use a terminal
+that supports tabs in order to do this. The terminals simply make your life easier as different
+logical activities will be performed in each one.
```

Comment:
"different logical actors have their own terminal in which to perform their actions"?